src/models/StationModels.tsx

```diff
@@ -0,0 +1,6 @@
+interface StationModel {
+    name: string;
+    crs: string;
+}
+
+export default StationModel;
```

Comment:
really nice you've made a models folder!

currently you've got an `export default StationModel;` here though the other models are exporting directly, in the main project we will have files within models that may contain multiple models for a particular section of the code so we usually want to be exporting directly e.g. `export interface Station Model` as that allows us to have multiple models in a file.

Though here looks like you're using one model per file so shouldn't matter too much either way, though we'd like consistency